src/index.ts

```diff
@@ -1,5 +1,10 @@
 import { Server } from "./Server";
 import { config } from "./config/configuration";
+import { IConfig } from '../src/config/IConfig'
+import { default as notFound } from './libs/routes/notFoundRoute';
+import { errorHandler } from './libs/routes/errorHandler';
 
 const serve = new Server(config);
 serve.bootstrap().run();
+
+export { IConfig, notFound, errorHandler };
```

Comment:
Should be export from root folder only like
 - IConfig should be exported from config folder.
 - notFoundRoute, errorHandler should be exported from libs folder only.
 - change notFound to notFoundRoute